conda-env/dev.yml

```diff
@@ -27,32 +26,32 @@ dependencies:
     # Documentation
     # ==================
     - sphinx=4.5.0
-    - sphinxcontrib-napoleon=0.7
     - sphinx-autosummary-accessors=2022.4.0
     - sphinx-book-theme=0.3.3
     - sphinx-copybutton=0.5.0
     - nbsphinx=0.8.9
-    - pandoc=2.19
+    - pandoc=2.19.2
+    - ipython=8.5.0 # Required for nbsphinx syntax highlighting
     # ==================
     # Quality Assurance
     # ==================
     # NOTE: If versions are updated, also update 'rev' in `.pre-commit.config.yaml`
-    - black=22.6.0
+    - black=22.10.0
     - flake8=5.0.4
-    - flake8-isort=4.2.0
+    - flake8-isort=5.0.0
     - isort=5.10.1
-    - mypy=0.971
+    - mypy=0.990
     - pre-commit=2.20.0
     - types-python-dateutil=2.8.19
     # ==================
     # Testing
     # ==================
-    - pytest=7.1.2
-    - pytest-cov=3.0.0
+    - pytest=7.2.0
+    - pytest-cov=4.0.0
     # ==================
     # Developer Tools
     # ==================
-    - matplotlib=3.5.2
-    - jupyterlab=3.4.5
+    - matplotlib=3.6.2
+    - jupyterlab=3.5.0
     - tbump=6.9.0
 prefix: /opt/miniconda3/envs/xcdat_dev
```

Comment:
Hey @xCDAT/core-developers, I recommend rebuilding your `xcdat_dev` env using the latest `conda-env/dev.yml` on `main`. Many dependencies have been bumped.

No package conflicts and everything is working the same as before.